CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the "silverfin-development-toolkit" extension will be documented in this file.
 
+## [1.20.2]
+
+- Reset silverfin-cli version
```

Comment:
Let's change to something like: `Revert silverfin-cli version to 1.31.0` as it introduces breaking changes related to liquid testing for account templates currently not supported in the extension`